src/main/mulmo/handler_contents.ts

```
@@ -95,3 +100,36 @@
+          return MulmoStudioContextMethods.resolveAssetPath(context, image.source.path);
+        }
+      })();
+      if (path && fs.existsSync(path)) {
+        const buffer = fs.readFileSync(path);
+        imageRefs[key] = buffer.buffer;
+      }
+    });
+  return imageRefs;
+};
+
+const mulmoImageRefecenceImage = () => {};
```

ESLint

[error] 'mulmoImageRefecenceImage' is assigned a value but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)